src/main/java/com/e451/service/CouponRedemptionService.java

```diff
@@ -0,0 +1,12 @@
+package com.e451.service;
+
+
+import com.e451.domain.CouponRedemption;
+import com.e451.models.CouponRedemptionRecord;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.domain.Page;
+
+public interface CouponRedemptionService {
+    Page<CouponRedemption> getCouponRedemptions(org.springframework.data.domain.Pageable pageable);
+    Converter<CouponRedemptionRecord, CouponRedemption> getCouponRedemptionRecordCampaignMembershipConverter();
```

Comment:
I think the name of the method should be getCouponRedemptionRecordCouponRedemptionConverter. I think the underlying property name is correct.